spec/notification.js

```diff
@@ -375,6 +375,19 @@ describe('notification api', function() {
           });
     });
 
+    it('should get all received texts', function(done) {
+
+      notifyAuthNock
+        .get('/v2/received-text-messages')
+        .reply(200, {"foo":"bar"});
+
+      notifyClient.getReceivedTexts()
+        .then(function(response){
+          expect(response.statusCode).to.equal(200);
+          done();
+        });
+    });
+
```

Comment:
should probably also add an `older_than` unit test for received texts similar to line 260 in this file